src/passes/RemoveUnusedBrs.cpp

```diff
@@ -1258,6 +1258,45 @@ struct RemoveUnusedBrs : public WalkerPass<PostWalker<RemoveUnusedBrs>> {
           tablify(curr);
           // Pattern-patch ifs, recreating them when it makes sense.
           restructureIf(curr);
+
+          // A potential optimization opportunity:
+          // Optimize block tails where a dropped `br_if`'s value is redundant
+          // when the br_if targets the block itself.
+          // consider for example:
+          // (block $block (result i32)
+          // ..
+          //   (drop
+          //     (br_if $block ;; <- MUST target parent $block
+          //       (value)
+          //       (condition)
+          //     )
+          //   )
+          //   (value) ;; <- MUST be same as br_if's value
+          // )
+          // =>
+          // (block $block (result i32)
+          // ..
+          //   (drop
+          //     (condition)
+          //   )
+          //   (value)
+          // )
+          const size_t size = curr->list.size();
```

Comment:
```suggestion
          size_t size = curr->list.size();
```
This is constant, but by convention we only use `const` where it is useful (for readability or optimization).